inc/Engine/Optimization/RUCSS/Admin/Subscriber.php

```diff
@@ -53,14 +54,8 @@ public static function get_subscribed_events() : array {
 		return [
 			'rocket_first_install_options'       => 'add_options_first_time',
 			'rocket_input_sanitize'              => [ 'sanitize_options', 14, 2 ],
-			'update_option_' . $slug             => [
-				[ 'clean_used_css_and_cache', 10, 2 ],
-				[ 'clean_used_css_directory_cpcss_disabled', 10, 2 ],
```

Comment:
I thought that we will keep that so we remove the directory completely when CPCSS is disabled!

Reply:
If I remember well, we disabled the file cleaning in case there is external cache which can be out of sync with WP Rocket and might lead to 404 pages.
I think @Tabrisrp can clarify what should happen when CPCSS is disabled.

Reply:
Yes, got it.
Thanks for clarification @crystinutzaa